cmd/kind/create/cluster/createcluster.go

```diff
@@ -47,6 +48,7 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().StringVar(&flags.Name, "name", "1", "the cluster context name")
 	cmd.Flags().StringVar(&flags.Config, "config", "", "path to a kind config file")
 	cmd.Flags().StringVar(&flags.ImageName, "image", "", "node docker image to use for booting the cluster")
+	cmd.Flags().BoolVar(&flags.Retain, "retain", false, "whether retain the broken nodes for debugging")
```

Comment:
tiny nit, maybe description should be: "retain nodes for debugging when cluster creation fails"? even if a node is not broken, this flag will retain all of them in the near future.

Reply:
ok, I will update it.